scripts/output/elements.js

```diff
@@ -1,35 +1,58 @@
-import { mkdirSync, writeFileSync } from 'node:fs'
-import { join as joinPath } from 'node:path'
-import { getSVGs, getDirname } from '../util/helpers.js'
-import chalk from 'chalk'
+import { mkdirSync, writeFileSync } from "node:fs";
+import { join as joinPath } from "node:path";
+import { getSVGs, getDirname } from "../util/helpers.js";
+import chalk from "chalk";
+import defaultIconDescriptions from "../../default-icon-descriptions.js";
 
-const __dirname = getDirname(import.meta.url)
-const icons = []
-const basepath = joinPath(__dirname, '../../elements/')
-mkdirSync(basepath, { recursive: true })
+const __dirname = getDirname(import.meta.url);
+const icons = [];
+const basepath = joinPath(__dirname, "../../elements/");
+mkdirSync(basepath, { recursive: true });
 
+// Create Elements Icon
 getSVGs().forEach(({ svg, name, size, filename, exportName }) => {
-  const attrs = Array.from(svg.attrs).map(attr => attr.name + `=` + `"` + attr.value + `"`)
-  const className = exportName
+  const iconNameCamelCase = exportName.replace(/Icon|\d+/g, "");
+  const titleMessage = defaultIconDescriptions[iconNameCamelCase.toLowerCase()];
+  const attrs = Array.from(svg.attrs).map(
+    (attr) => attr.name + `=` + `"` + attr.value + `"`
+  );
+  const { message, id, comment } = titleMessage || {};
+  const titleHtml = "<title>${title}</title>";
```

Comment:
Any reason to have this as a variable?

Reply:
No reason for me. I just implemented the same changes from the scripts/output/react.js-file. But I agree, it can be moved directly to the output-variable on line 37. :)

Reply:
I think I understand why `<title>${title}</title>` is in the titleHtml variable. When I try to move `<title>${title}</title>` to line 37 (`return html\`<svg ${attrs.join("")}><title>${title}</title>${svg.html}</svg>\`) I get a ReferenceError: "title is not defined". It seems that the title variable on line 35 doesn't have time to be defined. I'm guessing that it's expecting that the title-variable, which is defined on line 35, should be outside of the template-literal from the output-variable. I can either let it be as it is now with that `<title>${title}</title>` stays in the titleHtml-variable, or I move up line 36 to outside of the output-variable and then use the title-variable directly on line 37. Wdyt? :)

Reply:
> I can either let it be as it is now

Yeah, let's do that :)

Reply:
Ok I'll move the `title`variable up to outside of the output-variable then! :)

Reply:
hmm we seem to loose the i18n when I move the `title`variable up to outside of the output-variable. It is no longer reacting when I change the message in the /fi/messages.mjs-file and change the language for the html to "fi".